src/screens/sign-up/sign-up-screen.tsx

```diff
@@ -1,12 +1,160 @@
 import * as React from "react";
-import { Text, View } from "react-native";
-import { COMMON_STYLES } from "~theme/common-styles";
+import { useEffect, useState } from "react";
+import {
+  Text,
+  TouchableWithoutFeedback,
+  View,
+  Keyboard,
+  ScrollView,
+} from "react-native";
+import { COMMON_STYLES } from "../../theme/common-styles";
+import ButtonComponent from "./components/ButtonComponent";
+import FormFieldsComponent from "./components/FormFieldsComponent";
+import LogoComponent from "./components/LogoComponent";
+import axios from "axios";
+import Icon from "react-native-vector-icons/FontAwesome5";
+
+export interface UserDetailsProps {
+  firstName?: string;
+  lastName?: string;
+  fullName?: string;
+  email?: string;
+  phoneNumber?: string;
+  password?: string;
+  [key: string]: any;
+}
+
+export enum Position {
+  ABSOLUTE = "absolute",
+  RELATIVE = "relative",
+}
+
+interface StyleProps {
+  color: string;
+  top: number;
+  left: number;
+  position: Position;
+  borderRadius: number;
+  bgColor: string;
+  width: number;
+  height: number;
+  inputBorderRadius: number;
+  inputMarginTop: number;
+  backBtnLeft: number;
+}
 
 function SignUpScreen() {
+  const [userDetails, setUserDetails] = useState<UserDetailsProps>({
+    firstName: "",
+    lastName: "",
+    email: "",
+    phoneNumber: "",
+    password: "",
+  });
+  const [showForm, setShowForm] = useState<boolean>(true);
+  const [styles, setStyles] = useState<StyleProps>({
+    color: "",
+    top: 0,
+    left: 0,
+    position: Position.RELATIVE,
+    borderRadius: 0,
+    bgColor: "",
+    width: 0,
+    height: 0,
+    inputBorderRadius: 0,
+    inputMarginTop: 0,
+    backBtnLeft: 0,
+  });
+
+  const formFields = [
+    { name: "firstName", placeholder: "Enter first name" },
+    { name: "lastName", placeholder: "Enter last name" },
+    { name: "phoneNumber", placeholder: "Enter phone number" },
+    { name: "email", placeholder: "Enter email address" },
+    { name: "password", placeholder: "Enter your password" },
+  ];
+
+  const handleChangeUserDetails = (fieldName: string, value: string) => {
+    setUserDetails((prevDetails) => ({ ...prevDetails, [fieldName]: value }));
+  };
+
+  const handlePressSignUp = () => {
+    console.log("Sign up btn clicked");
```

Comment:
console.log statements should'nt be pushed

Reply:
All console.log statements have been removed as instructed